src/south/SQLDbToFile/SQLDbToFile.class.js

```diff
@@ -204,12 +233,13 @@ class SQLDbToFile extends ProtocolHandler {
 
   /**
    * Get new entries from MSSQL database.
-   * @param {Date} lastCompletedAt - The last completed date
-   * @returns {void}
+   * @param {Date} startTime - The start time
+   * @param {Date} endTime - The end time
+   * @returns {Promise<array>} - The new entries
    */
-  async getDataFromMSSQL(lastCompletedAt) {
+  async getDataFromMSSQL(startTime, endTime) {
     const adaptedQuery = this.query
-    this.logger.debug(`Executing "${adaptedQuery}" ${this.containsLastCompletedDate ? 'with' : 'without'} LastCompletedDate`)
+    this.logger.debug(`Executing "${adaptedQuery}" ${this.nrOfReplacements ? 'with' : 'without'} StartTime/EndTime/MaxReturnValues`)
```

Comment:
Can you replace this log by a more elaborate one ? Something like :
Executing "${adaptedQuery}" with StartTime = ${StartTime.toISOString()} (string formatted),  EndTime = ${EndTime.toISOString()} (string formatted), MaxReturnValues = ${MaxReturnValues}

Displaying only the variable used in the query.